Fabric.Terminology.Domain/Services/ValueSetService.cs

```diff
@@ -134,55 +135,78 @@ public bool NameIsUnique(string name)
                 return Attempt<IValueSet>.Failed(new ArgumentException(msg));
             }
 
-            var valueSetCodeItems = valueSetCodes as IValueSetCodeItem[] ?? valueSetCodes.ToArray();
-            if (!valueSetCodeItems.Any())
+            var setCodes = codeSetCodes as IValueSetCode[] ?? codeSetCodes.ToArray();
```

Comment:
I had changed the params to be `IReadOnlyCollection` and then got a bunch of comments from people who really felt it made the API harder to work with.  The general consensus was that the developer that is calling the code should be responsible for the enumerable being passed to the method - so I changed it back.